LibreNMS/Device/YamlDiscovery.php

```diff
@@ -32,6 +33,8 @@
 
 class YamlDiscovery
 {
+    private static $cache_time = 1800; // 30 minutes, Used for oid translation cache
```

Comment:
Is there a reason to only cache it for a short time?

Reply:
We want it to cover 1 discovery. And of course avoid caching for more than 24 hours (daily might come with new MIB files). So clearly we could go to 2 or 3 hours. More than that probably does not make sense, the source of the information must remain the MIB file. I'll got that route.